inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/reverse_sequence.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/reverse_sequence.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32,
+        InferenceEngine::Precision::FP16,
+        InferenceEngine::Precision::U8, //doesn't match the reference values
+        InferenceEngine::Precision::I8, //doesn't match the reference values
+        InferenceEngine::Precision::U16,
+        InferenceEngine::Precision::I32
+};
+
+const std::vector<int64_t> batchAxisIndices = { 0L };
+const std::vector<int64_t> seqAxisIndices = { 1L };
+
+const std::vector<std::vector<size_t>> inputShapes = { {3, 10} }; //, 10, 20
+const std::vector<std::vector<size_t>> reversSeqLengthsVecShapes = { {3} };
+std::vector<ngraph::helpers::InputLayerType> secondaryInputTypes = {
```

Comment:
Minor comment, new line between vectors will be more elegant

Reply:
Fixed.